isis/tests/FunctionalTestsCsminit.cpp

```diff
@@ -0,0 +1,342 @@
+#include "csminit.h"
+
+#include <QString>
+#include <iostream>
+
+#include "AlternativeTestCsmModel.h"
+#include "TestCsmPlugin.h"
+#include "Fixtures.h"
+#include "TestUtilities.h"
+#include "TestCsmModel.h"
+#include "StringBlob.h"
+#include "FileName.h"
+
+#include <nlohmann/json.hpp>
+using json = nlohmann::json;
+
+#include "gmock/gmock.h"
+
+using namespace Isis;
+
+static QString APP_XML = FileName("$ISISROOT/bin/xml/csminit.xml").expanded();
+
+class CSMPluginFixture : public TempTestingFiles {
```

Comment:
What's the rationale for putting this in the test instead of fixtures.cpp?

Reply:
The initial thought was that since it's only used in one place, leave it in the test. If/when it gets used for more than one test, we can move it into Fixtures.cpp.